### DISCUSSION TITLE
Welcome to the Grouparoo Forum!

### DISCUSSION THREAD

**Discussion by evantahler** [grouparoo/grouparoo#675] [Chat]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.  As a reminder, this forum is subject to our [Community Code of Conduct](https://www.grouparoo.com/docs/community/code-of-conduct)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by evantahler** [comment]
Hi! I'm Evan, the CTO and one of Grouparoo's founders.  I'm also active in open source over at the @actionhero project(s).  You can find me at @evantahler on Twitter, Github and other places.  I'm the one most likely to read and fix your Issues :D

Welcome to Grouparoo, we are glad you are here!

**Comment by bleonard** [comment]
Hi! I'm Brian, the CEO and co-founder of Grouparoo. I've been the CTO of TaskRabbit for a while and contributed to several open source projects through that. I tweet from @bleonard and am now writing blog posts on the Grouparoo [blog](https://www.grouparoo.com/blog).

Welcome!

**Comment by andyjih** [comment]
Hi everyone! I'm Andy, the COO and co-founder of Grouparoo. I've worked on products at companies such as Stripe, TaskRabbit, and Zynga throughout my career. You can find me @andyjih on Twitter and most other places. Looking forward to "meeting" you all!
